src/resources/styles.ts

```diff
@@ -34,36 +34,36 @@ export const haStyle = css`
   }
 
   h1 {
-    font-family: var(--paper-font-headline_-_font-family);
-    -webkit-font-smoothing: var(--paper-font-headline_-_-webkit-font-smoothing);
+    font-family: var(--ha-font-family-body);
+    -webkit-font-smoothing: var(--ha-font-smoothing);
     white-space: var(--paper-font-headline_-_white-space);
     overflow: var(--paper-font-headline_-_overflow);
     text-overflow: var(--paper-font-headline_-_text-overflow);
-    font-size: var(--paper-font-headline_-_font-size);
-    font-weight: var(--paper-font-headline_-_font-weight);
-    line-height: var(--paper-font-headline_-_line-height);
+    font-size: var(--ha-font-size-2xl);
+    font-weight: var(--ha-font-weight-normal);
+    line-height: var(--ha-line-height-condensed);
   }
 
   h2 {
-    font-family: var(--paper-font-title_-_font-family);
-    -webkit-font-smoothing: var(--paper-font-title_-_-webkit-font-smoothing);
-    white-space: var(--paper-font-title_-_white-space);
-    overflow: var(--paper-font-title_-_overflow);
-    text-overflow: var(--paper-font-title_-_text-overflow);
-    font-size: var(--paper-font-title_-_font-size);
-    font-weight: var(--paper-font-title_-_font-weight);
-    line-height: var(--paper-font-title_-_line-height);
+    font-family: var(--ha-font-family-body);
+    -webkit-font-smoothing: var(--ha-font-smoothing);
+    white-space: var(--ha-title-h2-white-space);
+    overflow: var(--ha-title-h2-overflow);
+    text-overflow: var(--ha-title-h2-text-overflow);
+    font-size: var(--ha-font-size-xl);
+    font-weight: var(--ha-font-weight-semibold);
+    line-height: var(--ha-line-height-normal);
   }
 
   h3 {
-    font-family: var(--paper-font-subhead_-_font-family);
-    -webkit-font-smoothing: var(--paper-font-subhead_-_-webkit-font-smoothing);
+    font-family: var(--ha-font-family-body);
+    -webkit-font-smoothing: var(--ha-font-smoothing);
     white-space: var(--paper-font-subhead_-_white-space);
     overflow: var(--paper-font-subhead_-_overflow);
     text-overflow: var(--paper-font-subhead_-_text-overflow);
```

Comment:
Are these missed? They are done for `h2`?

Reply:
They havn't been set before. I removed the var and the css attribute in h1 and h3